modules/osdk-cli-reference-new.adoc

```diff
@@ -22,6 +22,9 @@ The `operator-sdk new` command creates a new Operator application and generates
 |`--api-version`
```

Comment:
The `new` subcommand no longer exists in v0.19.x. We should remove docs about it.